contracts/contracts/subnet/SubnetActorManagerFacet.sol

```diff
@@ -213,10 +222,10 @@ contract SubnetActorManagerFacet is SubnetActorModifiers, ReentrancyGuard, Pausa
 
         if (!s.bootstrapped) {
             LibStaking.withdrawWithConfirm(msg.sender, amount);
-            return;
+            s.collateralSource.transferFunds(payable(msg.sender), amount);
+        } else {
+            LibStaking.withdraw(msg.sender, amount);
```

Comment:
This else statement should also contain `s.collateralSource.transferFunds(payable(msg.sender), amount);` no? I ran a test that unstakes when the subnet is bootstrapped and the validator didnt get their erc20 back

Reply:
so after the subnet is bootstrapped, that means any validator stake withdraw should not have occurred immediately because of safety issues. It has to go through the checkpointing and confirmed by the current commitee.

Reply:
@JulissaDantes check this out: https://github.com/consensus-shipyard/ipc/blob/6d2cc3a52451a04e31970484a482a6251b08ee8f/contracts/contracts/lib/LibStaking.sol#L667

It's buffered in a release queue. 

See tests: https://github.com/consensus-shipyard/ipc/blob/6d2cc3a52451a04e31970484a482a6251b08ee8f/contracts/test/integration/SubnetActorDiamond.t.sol#L1902

To withdraw stakes, a lot of checks have to go through.

Reply:
oh, I think I'm little lost here, 
- what is the `current commitee.`? 
- does that mean there is somewhere else in the code where it gets check if any tokens are owed to a validator? 
- or does that mean that at the checkpoint library this is handled where they check unstake request and perform the token transfers?

And Im not clear here, my understanding is that a `bootstrapped` subnet is an active subnet that has the minimum required collateral and validators amount to run. So I still think that there is a path for a validator to just want to unstake certain amount of their stake after a while, where there are several checkpoints being made, and they just wish to reduce their staked amount, although if this is handled elsewhere I understand why its not the case, depending on the answers to the previous questions I guess.